showkase-processor-testing/src/test/resources/ShowkaseProcessorTest/composable_function_inside_companion_object_with_preview_annotation_compiles_ok/output/ShowkaseMetadataShowkase_processor_testing.kt

```diff
@@ -17,7 +17,9 @@ public class ShowkaseMetadataShowkase_processor_testing {
     showkaseKDoc = "",
     enclosingClass = [ShowkaseClass::class],
     showkaseMetadataType = "COMPONENT",
-    isDefaultStyle = false
+    isDefaultStyle = false,
+    showkaseWidthDp = -1,
```

Comment:
xprocessing doesn't distinguish between default param values and custom values when retrieving values off an annotation, so these end up getting set, but the behavior is the same.